go.mod

```diff
@@ -3,6 +3,7 @@ module github.com/celestiaorg/test-infra
 go 1.19
 
 replace (
+	github.com/celestiaorg/celestia-node => github.com/derrandz/celestia-node v0.6.3-0.20230311140423-894e3a6ec124
```

Comment:
what is the reason behind using a fork?

Reply:
to use some extra metrics for testing, will remove now